File: packages/prism-nexus/ios/Hydrator/HydratorConstants.swift
import Foundation

enum HydratorConstants {
  static let ttlSeconds: TimeInterval = 300
  static let ttlMs: UInt64 = 300_000
  static let maxStorageBytes = 50 * 1024 * 1024
  static let kidV1 = "v1"
  static let cacheDirName = "Hydrator"
  static let chunkSize = 4096
  static let chunkThreshold = 20 * 1024
  static let gcmTagBytes = 16
  static let reassemblyTimeoutMs: UInt64 = 30_000
  static let aesKeyTag = "com.prismnexus.hydrator.aes.v1"
  static let ed25519KeyTag = "com.prismnexus.hydrator.ed25519.v1"
}

File: packages/prism-nexus/android/src/main/java/com/prismnexus/hydrator/HydratorConstants.kt
package com.prismnexus.hydrator

object HydratorConstants {
  const val TTL_SECONDS = 300L
  const val TTL_MS = 300_000L
  const val MAX_STORAGE_BYTES = 50 * 1024 * 1024
  const val KID_V1 = "v1"
  const val CACHE_DIR_NAME = "Hydrator"
  const val CHUNK_SIZE = 4096
  const val CHUNK_THRESHOLD = 20 * 1024
  const val GCM_TAG_BYTES = 16
  const val REASSEMBLY_TIMEOUT_MS = 30_000L
  const val AES_KEY_ALIAS = "com.prismnexus.hydrator.aes.v1"
  const val ED25519_PREFS = "com.prismnexus.hydrator.ed25519.v1"
}
